kex-runner/src/main/kotlin/org/vorpal/research/kex/launcher/ConcolicLauncher.kt

```diff
@@ -36,6 +43,11 @@ class ConcolicLauncher(classPaths: List<String>, targetName: String) : KexAnalys
                 InstructionConcolicChecker.run(context, setOfTargets)
             }
         }
+
+        if (minimization) {
```

Comment:
Can we somehow reuse the coverage information from `Minimizer`? So that we don't have to rerun whole test suite again? Maybe we can create some alternative `minimizeAndReportCoverage` implementation that would run the minimization and reuse its coverage info for reporting the coverage to the end user.

Reply:
I see next desicions:

1. Minimizer.execute() can probably return this information for example in CommonCoverageInfo format
But for me it seems a little bit messy because it looks as not obvious behaviour.

2. Minimizer can have same logic inside of it. for example it will have method minimizer.ReportCoverage(). And it will reuse information.
looks as best option for me

Reply:
Yes, I think option 2 is good